### PR TITLE
build the sections 'works' and 'about me' of mobile website

### PR DESCRIPTION
**Pull request by fahdrahali**
**build the middle section of the mobile website, there are two sections, the first is "works" section and the second is "about me section", and I use:**
- Semantic Html
- CSS Grid
- FlexBox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,9 +30,72 @@ <h1 class="box">Hey there. I'm Fahd <br> <span class="second-title">I'm a softwa
...
+                <li><h6 class="badge bg-gray oil ruby-rails-badge">Ruby on rails</h6></li>
+                <li><h6 class="badge bg-gray oil css-badge">css</h6></li>
+                <li><h6 class="badge bg-gray oil js-badge">JavaScript</h6></li>
+                <li><h6 class="badge bg-gray oil html-badge">html</h6></li>
+            </ul>
+            <button class="btn-green btn-projects" type="button">See Project</button>
+        </div>
+    </section>
+    <section class="about">
+        <h1 class="title">About Me</h1>
```
Comment (mateo951) [inline index.html:56]: As I already mentioned before, having only 1 `h1` is important for semantics and SEO reasons! Kindly change this heading 🤓

File: index.html [hunk 2]
```diff
@@ -30,9 +30,72 @@ <h1 class="box">Hey there. I'm Fahd <br> <span class="second-title">I'm a softwa
...
+            <div class="flex-card">
+                <ul class="about-card">
+                    <li><img class="icon-card" src="./assets/images/icon-languages.png" alt="languages"></li>
+                    <li ><h1 class="title-card">Languages</h1></li>
+                    <li class="badge-list-about"><ul class="badge-list">
+                        <li><h6 class="badge bg-white green js-badge">JavaScript</h6></li>
+                        <li><h6 class="badge bg-white green ruby-badge">Ruby</h6></li>
+                        <li><h6 class="badge bg-white green css-badge">css</h6></li>
+                        <li><h6 class="badge bg-white green html-badge">html</h6></li>
+                    </ul></li>
```
Comment (micheaol) [inline index.html:69]: Please ensure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. Kindly remove `h6` tag from the `li` to make it semantically correct.

File: index.html [hunk 3]
```diff
@@ -30,9 +30,72 @@ <h1 class="box">Hey there. I'm Fahd <br> <span class="second-title">I'm a softwa
...
+            <div class="flex-card">
+                <ul class="about-card">
+                    <li><img class="icon-about-card" src="./assets/Icons/icon-frameworks.png" alt="Frameworks"></li>
+                    <li ><h1 class="title-card">Frameworks</h1></li>
+                    <li class="badge-list-about">
+                      <ul class="badge-list">
+                        <li><h6 class="badge bg-white green js-badge">Bootstrap</h6></li>
+                        <li><h6 class="badge bg-white green ruby-rails-badge">Ruby on rails</h6></li>
+                        <li><h6 class="badge bg-white green rspec-badge">RSpec</h6></li>
+                    </ul></li>
```
Comment (micheaol) [inline index.html:82]: The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`. Kindly remove `h6` tag from the `li` to make it semantically correct.

File: index.html [hunk 4]
```diff
@@ -30,9 +30,72 @@ <h1 class="box">Hey there. I'm Fahd <br> <span class="second-title">I'm a softwa
...
+            <ul class="badge-list">
+                <li><h6 class="badge bg-gray oil ruby-rails-badge">Ruby on rails</h6></li>
+                <li><h6 class="badge bg-gray oil css-badge">css</h6></li>
+                <li><h6 class="badge bg-gray oil js-badge">JavaScript</h6></li>
+                <li><h6 class="badge bg-gray oil html-badge">html</h6></li>
+            </ul>
+            <button class="btn-green btn-projects" type="button">See Project</button>
+        </div>
+    </section>
+    <section class="about">
```
Comment (mateo951) [inline index.html:55]: This section needs to have the background image (decorations are always background images, not img tags) as shown in Figma's design 

Captura

Kindly add the background to your app to meet the project requirements 📝

File: style.css [hunk 5]
```diff
@@ -102,16 +97,249 @@
...
+    text-align: center;
+    letter-spacing: 0.03em;
+  }
+
+  .about {
+    display: grid;
+    grid-template-columns: auto;
+    justify-content: center;
+    margin-top: 40px;
+  }
```
Comment (micheaol) [inline style.css:274]: Please ensure that the content of the `About section` is aligned with flexbox according to the requirement.

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @fahdrahali, 

📝👌 **Your project highlights** 🔥😎 
- Linters are on point ✔
- Great use of GitHub flow 👌

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 6]
```diff
@@ -30,9 +30,72 @@ <h1 class="box">Hey there. I'm Fahd <br> <span class="second-title">I'm a softwa
...
+        <div class="card">
+            <div class="img-card"></div>
+            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="badge-list">
+                <li><h6 class="badge bg-gray oil ruby-rails-badge">Ruby on rails</h6></li>
+                <li><h6 class="badge bg-gray oil css-badge">css</h6></li>
+                <li><h6 class="badge bg-gray oil js-badge">JavaScript</h6></li>
+                <li><h6 class="badge bg-gray oil html-badge">html</h6></li>
+            </ul>
+            <button class="btn-green btn-projects" type="button">See Project</button>
```
Comment (micheaol) [inline index.html:52]: Please ensure that the `works` section has six (6) cards according to the design. Please check [HERE](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) for help.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @fahdrahali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @fahdrahali, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly add more cards in your Works section according to the design.

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**